### PR TITLE
Modify docstrings

### PR DESCRIPTION
**Pull request by kmorhun**
This Pull Request:
- Makes minor changes to the main `README.md` to clarify how to create a conda environment on first installing the program.
- Updates docstrings in `grid.py`, `pednet.py`, `raster.py`, `tile.py` to be more consistently formatted, and prepare them for future sphinx and ReadTheDocs documentation

### CODE REVIEW

File: src/tile2net/raster/pednet.py [hunk 1]
```diff
@@ -87,8 +97,10 @@ def find_medianisland(self, swp, cwp, max_area=230):
     def validate_linemerge(self, merged_line):
         # from topojson https://github.com/mattijn/topojson/commit/cdc059bae53f3f5cfe882527e5d34e671f80173e
         """
-        Return list of linestrings. If the linemerge was a MultiLineString
-        then returns a list of multiple single linestrings
+        Return
+        ------
+        list[shapely.geometry.LineString]
```
Comment (dhodcz2) [inline src/tile2net/raster/pednet.py:102]: Returns
--------

File: src/tile2net/raster/tile.py [hunk 2]
```diff
@@ -151,8 +157,8 @@ def tile2poly(self, *bounds):
         return poly
 
     def tile2gdf(self, *bounds):
-        """Create a tile GeoDataFrame
-        Returns (GeoDataFrame): A GeoDataFrame of a single tile
+        """Create a tile :class:`GeoDataFrame`
+        Returns (:class:`GeoDataFrame`): A :class:`GeoDataFrame` of a single tile
```
Comment (dhodcz2) [inline src/tile2net/raster/tile.py:161]:
```suggestion
        Returns
        -------
        :class:`GeoDataFrame`
            A GeoDataFrame of a single tile.

```

File: src/tile2net/raster/tile.py [hunk 3]
```diff
@@ -306,17 +321,23 @@ def map_features(self, src_img, img_array=True):
...
+        gdf: :class:`GeoDataFrame`
+            The :class:`GeoDataFrame` to clip
+        spatial_index: :class:`GeoDataFrame.sindex`
+            The spatial index to clip the :class:`GeoDataFrame` against
+        crs: int
+            The desired coordinate reference system to save the network polygon with.
 
+        Returns
+        -------
+        :class:`GeoDataFrame` | int
```
Comment (dhodcz2) [inline src/tile2net/raster/tile.py:339]:
```suggestion
        :class:`GeoDataFrame` or int
```
numpy docstring style uses "or" instead or "|":
```    Returns
    -------
    out : ndarray or scalar
        Output array, element-wise comparison of `x1` and `x2`.
        Typically of type bool, unless ``dtype=object`` is passed.
        This is a scalar if both `x1` and `x2` are scalars.
```

File: src/tile2net/raster/tile.py [hunk 4]
```diff
@@ -333,10 +354,11 @@ def get_region(self, gdf: gpd.GeoDataFrame, spatial_index, crs=3857):
...
     @staticmethod
     def preds_to_binary(pred_arr, channel_scaling=None, bg_threshold=0):
-        """From Solaris
+        """*Adopted from the Solaris library to overcome dependency issues*
+
         Convert a set of predictions from a neural net to a binary mask.
 
-        Arguments
+        Parameters
         ---------
```
Comment (dhodcz2) [inline src/tile2net/raster/tile.py:362]:
```suggestion
        ----------
```

**Review by dhodcz2 — CHANGES_REQUESTED** [review]
Have you been using GitHub CoPilot or another sort of LLM autocomplete? It would greatly accelerate this task if you have not already been using such a thing. [GitHub CoPilot is free for students.](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-global-campus-for-students/apply-to-github-global-campus-as-a-student)

**Review by dhodcz2 — APPROVED** [review]
Looks great! Thank you for your time.

**Comment by Mary-h86** [comment]
Approved, thank you both @kmorhun and @dhodcz2 for the through and prompt exchange.

**Pull request merged**